### ISSUE TITLE
Unable to use consecutive dynamic environment variables

### ISSUE THREAD

**Issue by tesla232** [mikefarah/yq#970]
I am unable to use consecutive dynamic environment variables. For instance:

```
dancer="tiny" user="two" yq eval -n '.[env(dancer)].seperator.[env(user)]="blank"'
```
returns
```
tiny:
  seperator:
    two: blank
```
Which is what you would expect. However the command:
```
dancer="tiny" user="two" yq eval -n '.[env(dancer)].[env(user)]="blank"'
```
returns
```
Error: Bad expression, please check expression syntax
```

Version of yq: 4.13.4
Operating system: Mac
Installed via: Homebrew

**Comment by mikefarah** [comment]
Looks like a bug, as a work around you can:

```
dancer="tiny" user="two" yq eval -n '(.[env(dancer)] | .[env(user)])="blank"'
```

**Comment by mikefarah** [comment]
Fixed in 4.13.15
